src/main/resources/migrations/00058_make_key_on_metadata_key_unique.sql

```diff
@@ -0,0 +1,5 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_unique_constraint_on_key_on_metadata_key
+
+ALTER TABLE metadata_key ADD CONSTRAINT key_unique_key UNIQUE (key);
```

Comment:
It is tricky with the naming for this. May be the below `metadata_key_unique_key` or `metadata_key_unique` ?
 ```suggestion
ALTER TABLE metadata_key ADD CONSTRAINT metadata_key_unique_key UNIQUE (key);
```